packages/core/src/StateNode.ts

```diff
@@ -120,6 +120,8 @@ export class StateNode<
    */
   public order: number = -1;
 
+  public __xstatenode: true = true;
```

Comment:
just wondering - any particular reason why this has been added to StateNode? The main reason for this property was to check if something is a machine or not, do we need to check anywhere if something is a StateNode?

Reply:
Accidental addition. Will remove

Reply:
```suggestion
```